examples/hello.rs

```diff
@@ -5,35 +5,17 @@
 #![no_main]
 #![no_std]
 
-#[macro_use]
 extern crate cortex_m_rt as rt;
 extern crate cortex_m_semihosting as sh;
 extern crate panic_semihosting;
 extern crate stm32f103xx_hal;
 
 use core::fmt::Write;
+use rt::entry;
 
-use rt::ExceptionFrame;
-use sh::hio;
-
-entry!(main);
-
+#[entry]
 fn main() -> ! {
-    let mut hstdout = hio::hstdout().unwrap();
-
+    let mut hstdout = sh::hio::hstdout().unwrap();
     writeln!(hstdout, "Hello, world!").unwrap();
-
     loop {}
 }
-
-exception!(HardFault, hard_fault);
```

Comment:
Hm, you just removed the exception handlers here. Is that on purpose?

Reply:
Yes, that's a minimal exemple, using thé default zones.